### PR TITLE
#32

### PR DESCRIPTION
**Pull request by filiptrenuzh**

### CODE REVIEW

**Review by otopalak — CHANGES_REQUESTED** [review]
Please delete this space

File: src/index.js [hunk 1]
```diff
@@ -9,4 +9,5 @@ import App from "./App";
  * Applications built with just React usually have a single root DOM node.
  * More: https://reactjs.org/docs/rendering-elements.html
  */
+
```
Comment (otopalak) [inline src/index.js:12]: test

**Review by otopalak — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (filiptrenuzh) [reply src/index.js:12]: dä space bruchts

**Review by otopalak — APPROVED** [review]
gut

**Pull request merged**